layouts/shortcodes/observability_pipelines/log_source_configuration/logstash.md

```diff
@@ -0,0 +1,14 @@
+To configure Logstash to send logs to the Observability Pipelines Worker, use the following output configuration:
+
+```
+output {
```

Comment:
This indentation seems strangely large, but I'm not sure how these are typically formatted